src/containerapp-preview/README.rst

```diff
@@ -0,0 +1,5 @@
+Microsoft Azure CLI 'containerapp-preview' Extension
+==========================================
+
+This package is for the 'containerapp-preview' extension.
+i.e. 'az containerapp-preview'
```

Comment:
I think we can explain the role of `containerapp-preview` and its dependency on `containerapp` in the `README.rst`, so that more customers can understand it's usage. 
It would be even better if relevant concept documents could be linked here, which can increase the visibility of the concept documents.